com.endlessm.HackSoundServer.json.in

```diff
@@ -0,0 +1,28 @@
+{
+    "app-id": "com.endlessm.HackSoundServer",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
+    "sdk": "org.gnome.Sdk",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=session-bus",
+        "--socket=pulseaudio",
+        "--own-name=com.endlessm.HackSoundServer"
```

Comment:
I don't think this is necessary, applications are always allowed to own their own name.